tools/sgtrivy/tools.go

```diff
@@ -0,0 +1,99 @@
+package sgtrivy
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+//go:embed trivyignore
+var DefaultConfig []byte
+
+const (
+	version = "0.43.1"
+	name    = "trivy"
+)
+
+func defaultConfigPath() string {
+	return sg.FromToolsDir(name, ".trivyignore")
+}
+
+// CheckTerraformCommand checks terraform configuration on the given dir
+// for any known security misconfigurations.
+func CheckTerraformCommand(ctx context.Context, dir string) *exec.Cmd {
+	args := []string{
+		"config",
+		"--exit-code",
+		"1",
+		dir,
+	}
+
+	return Command(ctx, args...)
+}
+
+// Command returns a `trivy` *exec.Cmd.
+// It includes a flag to use a default .trivyignore.yaml which can be
+// overridedn by setting a .trivyignore.yaml in the git root.
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
```

Comment:
The `Command` function should always return a pure command without any alterations - but if we move the `.trivyignore` stuff up to the helper above, then the helper is a bit more justified!

Reply:
Make sense but the `.trivyignore` file isn't necessarily specific to Terraform so it wouldn't quite make sense to put it in the helper. I tried following the same pattern we have in other packages though I see now that mostly we do this in the `Run()` APIs which we don't have here.

If we remove it from here then I don't think it makes sense to supply the file if the caller has to write the logic to use.

In such case I think it is better to keep the Terraform helper and wire it there. I'll change the PR

Reply:
Sounds good!